index.html

```diff
@@ -0,0 +1,186 @@
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header class="toolbar_container">
+			<a href="#" class="Title_logo">My Logo</a>
+			<nav>
+				<img src="./images/i.png"  class="logo" alt="Icon menu">
+			</nav>
+		</header>
+
+		<main class="main_container">
+			<section class="text_container">
+				<h1 class="paragraphe">Hey There. <br/>I’m Leader.</h1>
+				
+				<h4>I'm a Software Developer</h4>
+				<p class="para">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+					don’t hesitate to contact me.</p>
+			</section>
+			
+			<section class="icons-container">
+				<ul class="icons-containers">
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon -GitHub.png"/></li>
+					<li><img src="./images/Icon - Linkedin.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Enabled.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Twitter.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Medium.png"/></li>	
+				</ul>				
+			</section>
+		</main>
+	</section>
+	
+	<section class="second_container">
+		<div class="header_title">
+		<h2>My Recent Works</h2>
+		<p class="line"></p>
+		</div>
+		<div class="recent_work_container">
+			<img src="./images/imgs.png" alt="works_section">
+			<div class="recent_work_container_text">
+				<h3>Multi-Post Stories</h3>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+			<div class="butons_container">
+				<button type="button">CSS</button>
+				<button type="button">HTML</button>
+				<button type="button">Boostrap</button>
+				<button type="button">Ruby</button>
+			</div>
+			<button  type="button" class="project_btn">See project</button>
+			</div>
+		</div>
+	</section>
+	<section>
+		<div class="container">
+			<div class="box">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button  type="button"class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/229276880-9822bf66-b132-4fec-b274-735324ead2ee.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/229277048-e06ba9fa-5bd1-4b2d-936f-55f3d49a2dd9.png)

Kindly adjust the spacing of this section to match the template on Figma 💪

According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) It is important that we stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Reply:
- [ ] This suggestion will help you to have a nice aesthetic piece of art if implemented. Kindly ensure that your spacing, font-sizes, and font-weights are as we have in the Figma design